### PR TITLE
create pages

### PR DESCRIPTION
**Pull request by AlexeyBrambalov**
result of the group coding on Sunday

### CODE REVIEW

File: frontend/src/components/ArtistPage/ArtistPage.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export default function ArtistPage({categories}) {
+    console.log(categories)
```
Comment (dahfool) [inline frontend/src/components/ArtistPage/ArtistPage.js:4]: remove `console.log` please

File: frontend/src/components/ArtistPage/ArtistPage.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export default function ArtistPage({categories}) {
```
Comment (dahfool) [inline frontend/src/components/ArtistPage/ArtistPage.js:3]: Please use es6 arrow functions to create component
`export const ArtistPage  = () => {}`

**Comment by dahfool** [comment]
Is the linting working ?

**Comment by dahfool** [comment]
@AlexeyBrambalov Have you used `prop-types`

File: frontend/src/components/ArtistPage/ArtistPage.js (hunk 2 above)
Comment (AlexeyBrambalov) [reply frontend/src/components/ArtistPage/ArtistPage.js]: No, I need to pull updated version.  I know what is prop-types, but never used them

Comment (dahfool) [reply frontend/src/components/ArtistPage/ArtistPage.js]: Please try to implement `prop-types`, i will help if you are struggling.

Comment (dahfool) [reply frontend/src/components/ArtistPage/ArtistPage.js]: On the ArtistsPage and ArtistPage page.

Comment (dahfool) [reply frontend/src/components/ArtistPage/ArtistPage.js]: Good work on the code tidying.

Comment (AlexeyBrambalov) [reply frontend/src/components/ArtistPage/ArtistPage.js]: done

**Pull request merged**